Source/Fuse.Controls.Native/DarkMode.uno

```diff
@@ -58,6 +58,10 @@ namespace Fuse.Controls.Native.iOS
 		</App>
 		```
 	*/ 
+	[ForeignInclude(Language.Java,
+		"java.lang.Runnable",
+		"android.content.res.Configuration",
+		"android.app.Activity")]
 	public class DarkMode : NativeEventEmitterModule
```

Comment:
This code is from the other PR, but on a second thought, I think we should drop `public` here because we don't want users to use this class directly.

Reply:
I needed it public to be able to call the function changeDarkMode function from external Objective-C file

Reply:
In theory that should still work without public, or we have found a bug in the compiler I think. If you get compiler errors without public it's probably fine to keep it public.

Reply:
yeah, I had compiler errors of it not finding the class

Reply:
I see, then public it is. :)